README.md

```diff
@@ -141,6 +141,7 @@ send_to=[optional] developer@mail.com
 [email-ses]
 aws_access_key_id=[!] [optional] token
 aws_secret_access_key=[!] [optional] token
+aws_session_token=[!] [optional] token
```

Comment:
Should the description above be updated to reflect that these are indeed required for production? Are we using these in any other environments (and if not, does this leave a path for deploying untested code?)

Reply:
Good question. They're "optional" in general sense because, if an environment using using SMTP, then they aren't needed.
However, I'll update our `settings.production-example.ini` template to be clear that for _our_ purposes, they're required.
I'll also clarify that `aws_session_token` is definitely not wanted for prod.

> Are we using these in any other environments (and if not, does this leave a path for deploying untested code?)

The key ID and access key are needed in (our) prod, and all three are needed in local dev if targeting SES.